demisto_sdk/commands/common/git_util.py

```diff
@@ -596,9 +596,9 @@ def find_primary_branch(repo: Repo) -> str:
                 return ""
             for current_remote_ref in current_remote.refs:
                 current_remote_ref_str = str(current_remote_ref)
-                if current_remote_ref_str == "origin/main":
+                if current_remote_ref_str == f"{DEMISTO_DEFAULT_REMOTE}/main":
                     return "main"
-                elif current_remote_ref_str == "origin/master":
+                elif current_remote_ref_str == f"{DEMISTO_DEFAULT_REMOTE}/master":
                     return "master"
```

Comment:
shouldn't we just replace this with `return GIT_PRIMARY_BRANCH`?

Reply:
And if the user has multiple remotes? it seems the code supports it

Reply:
you can use `f"{DEMISTO_DEFAULT_REMOTE}/{self.find_primary_branch()"

or something like that, and then you refer to the remote as well

Reply:
@eyalpalo I would add a check that if the default branch is not `master` or `main` to take the `f"{DEMISTO_DEFAULT_REMOTE}/GIT_PRIMARY_BRANCH"`

this is very rare case though, but can be covered now